.claude/queues/scout/README.md

```
@@ -0,0 +1,3 @@
+# Scout Queue
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)